scripts/pr_review_test.py

```
@@ -2163,14 +2163,31 @@ def test_exhausted_budget_is_partial_and_never_clean(self) -> None:
+        Raising discovery effort without raising this cap lets the call expire
+        before it emits a single finding, and an empty discovery pass publishes
+        as a clean review rather than as a failure. That is the fail-open
+        direction, so the two constants have to move together.
+        """
+        discovery_effort = pr_review.reasoning_for_phase("default", "review-chunk-1")
+        if discovery_effort in {"high", "xhigh"}:
+            self.assertGreaterEqual(
+                pr_review.DEFAULT_MAX_COMPLETION_TOKENS,
+                pr_review.JUDGE_MAX_COMPLETION_TOKENS,
+                "discovery runs at %s reasoning but its output allowance is below the "
+                "budget the judge already needed at high reasoning" % discovery_effort,
```

Ruff (0.16.3)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)